Makefile

```diff
@@ -9,51 +9,47 @@ DEFINES := -DVERSION=\"$(VERSION)\"
 CXXFLAGS += $(FLAGS) $($(BUILD)_FLAGS) $(DEFINES)
 LDLIBS += -lpthread -lusb-1.0
 SOURCES := $(wildcard *.cpp) $(wildcard */*.cpp)
-OBJECTS := $(SOURCES:.cpp=.o) firmware.o
+OBJECTS := $(SOURCES:.cpp=.o)
 DEPENDENCIES := $(SOURCES:.cpp=.d)
 
-DRIVER_URL := http://download.windowsupdate.com/c/msdownload/update/driver/drvs/2017/07/1cd6a87c-623f-4407-a52d-c31be49e925c_e19f60808bdcbfbd3c3df6be3e71ffc52e43261e.cab
-FIRMWARE_HASH := 48084d9fa53b9bb04358f3bb127b7495dc8f7bb0b3ca1437bd24ef2b6eabdf66
-
 PREFIX := /usr/local
 BINDIR := $(PREFIX)/bin
+DATADIR := $(PREFIX)/share/xow/firmware
```

Comment:
I think putting the firmware into /lib/firmware would be more correct

Reply:
Either one works for me. As long as the maintainers are happy I'd go with that.

Reply:
to quote Medusalix:
```
If we can't get the firmware into linux-firmware we can just provide a script that downloads the firmware and puts it into /lib/firmware automatically.
```

Reply:
Yup just saw the thread on discord (xexaxo here). Should be make it `/lib/firmware/microsoft/` or there are other suggestions?

Reply:
I think putting it in `/lib/firmware/xow_dongle.bin` should be fine because it's a MediaTek firmware modified by Microsoft.

Reply:
Ack, works for me.